docs/oauth.md

```diff
@@ -116,44 +163,46 @@ This set of OAuth Config capabilities is not shared with the regular connector c
 
 | Capability Name  | Description | Default | Recommendation |
 | ----  | ------- | --------- | ----------- |
-| OAUTH_CAP_SUPPORTS_CUSTOM_DOMAIN | Whether your OAuth provider supports custom domain. i.e. OAuth endpoint host is not fixed. | false | - |
+| OAUTH_CAP_SUPPORTS_CUSTOM_DOMAIN | Enable this if your IDP has different URLs. The OAuth config will specify the relative paths for the URLs, and the end user will have to provide the IDP hostname when creating a new connection. | false | - |
```

Comment:
Another limitation for byo v1 is that customers CANNOT enable OAUTH_CAP_SUPPORTS_CUSTOM_DOMAIN for customer provided oauthConfig.xml files, i.e we don't support custom domain/instanceurl for byo idps currently. We need to mention this, otherwise custom oauth configs will fail to load as well.

OAUTH_CAP_SUPPORTS_CUSTOM_DOMAIN  only works for embedded idps.